demo/d2l-input-search.html

```diff
@@ -4,66 +4,90 @@
+		instantiated, otherwise the CSS @apply mixin polyfill won't be
+		able to expand the underlying CSS custom properties.
+		See: https://github.com/Polymer/polymer-modulizer/issues/154
+		-->
+	<script type="module">
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<custom-style>
+			<style is="custom-style" include="demo-pages-shared-styles"></style>
+		</custom-style>`;
+
+document.body.appendChild($_documentContainer.content);
+</script>
+		<script type="module">
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<custom-style include="d2l-typography">
+			<style is="custom-style" include="d2l-typography"></style>
+		</custom-style>`;
+
+document.body.appendChild($_documentContainer.content);
+</script>
+		<script type="module">
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<style>
+			html {
+				font-size: 20px;
+			}
+		</style>`;
+
+document.body.appendChild($_documentContainer.content);
+</script>
 	</head>
 	<body unresolved class="d2l-typography">
-		<div class="vertical-section-container centered">
-
-			<h3>Search Input</h3>
-			<demo-snippet>
-				<template>
-					<d2l-input-search
-						label="Search"
-						value="apples"
-						placeholder="Search for some stuff">
-					</d2l-input-search>
-				</template>
+		<script type="module">
+const $_documentContainer = document.createElement('template');
+
+$_documentContainer.innerHTML = `<div class="vertical-section-container centered">
+
+			<h3>Search Input</h3>
+			<demo-snippet>
+				<template>
+					<d2l-input-search label="Search" value="apples" placeholder="Search for some stuff">
+					</d2l-input-search>
+				</template>
 			</demo-snippet>
 
 			<h3>Search Input (RTL)</h3>
 			<demo-snippet dir="rtl">
 				<template>
-					<d2l-input-search
-						label="Search"
-						value="apples"
-						placeholder="Search for some stuff">
+					<d2l-input-search label="Search" value="apples" placeholder="Search for some stuff">
 					</d2l-input-search>
 				</template>
 			</demo-snippet>
 
 			<h3>Search Input (No Clear)</h3>
 			<demo-snippet>
 				<template>
-					<d2l-input-search
-						label="Search"
-						value="Clear button will never appear"
-						placeholder="Search for some stuff"
-						no-clear>
+					<d2l-input-search label="Search" value="Clear button will never appear" placeholder="Search for some stuff" no-clear="">
 					</d2l-input-search>
 				</template>
-			</demo-snippet>
-
-		</div>
-		<script>
-			document.body.addEventListener('d2l-input-search-searched', function(e) {
-				// eslint-disable-next-line no-console
-				console.log(e);
-			});
-		</script>
-	</body>
-</html>
+			</demo-snippet>
+
+		</div>`;
+
+document.body.appendChild($_documentContainer.content);
+</script>cument.body.addEventListener('d2l-input-search-searched', function(e) {

```

Comment:
Something really strange has happened here.  Have not been able to figure this out yet.

Reply:
I've seen weirdness like this before if the line endings weren't LF

Reply:
@dlockhart @awikkerink : i've poked around in the code and i don't see the source of this.  Is this sort of thing something you've come across before?  The input (html) looks pretty clean from what i can tell.

Reply:
Thanks @margaree .  I will look into that.

Reply:
You were right about the line-endings!